www/gatsby-node.js

```diff
@@ -46,6 +46,12 @@ const slugToAnchor = slug =>
 exports.createPages = ({ graphql, actions, reporter }) => {
   const { createPage, createRedirect } = actions
 
+  createRedirect({
+    fromPath: `/docs/themes/api-reference`, // Merged Component CSS and CSS Modules
+    toPath: `/docs/themes-api/`,
```

Comment:
It looks like the path down below is `/theme-api/` rather than `/themes-api/`, not sure if that's intentional?

Reply:
oh, nope oversight. gracias.

Reply:
That's the only thing that needs to be updated, IMO!

Reply:
fixed!